### PR TITLE
Photo page

### PR DESCRIPTION
**Pull request by irisliu77**

### CODE REVIEW

**Comment by irisliu77** [comment]
Ready for review!

**Comment by joanchenx** [comment]
Approved from my end. As a note, it'd be good to add to the photo picker where if you search for the empty string, it'll show all your photos again. Or you can add a 'Clear Filters' button of some sort.

**Comment by irisliu77** [comment]
ready for re-review!

File: portfolio/src/main/webapp/photo.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                </nav>
+            </div>
+            <div class="hero-body">
+                <div class="columns is-centered">
+                    <div class="column is-one-third">
+                        <div class="field">
+                            <div class="control">
+                                <input class="input" id="photo-input" type="text" placeholder="Try portrait, cityview, arts...">
+                                <button class="button" id="filter-button">Filter</button>
+                                <button class="button" id="unfilter-button">Unfilter</button>
```
Comment (anbenson) [inline portfolio/src/main/webapp/photo.html:53]: The unfilter text is too big for its container - it's extending outside of the button, just FYI.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,13 +13,36 @@
 // limitations under the License.
 
 $(document).ready(function() {
+    $('#unfilter-button').css("display","none");
```
Comment (anbenson) [inline portfolio/src/main/webapp/script.js:16]: Would recommend that you don't set this initially to invisible from JS, but from CSS instead. That way if there's a problem loading JS, you won't have two buttons.

File: portfolio/src/main/webapp/photo.html (hunk 1 above)
Comment (irisliu77) [reply portfolio/src/main/webapp/photo.html:53]: solved

**Pull request merged**